packages/ipa-certified-attributes-importer/src/services/openDataExtractor.ts

```diff
@@ -211,21 +207,6 @@ export async function getAllInstitutions(
       return accumulator;
     }
 
-    const digitalAddress = extractor("Mail1", z.string());
```

Comment:
these filters should not be necessary anymore, as we import data only for tenants in the platform (thus already onboarded).
cc @beetlecrunch